### DISCUSSION TITLE
连接报错

### DISCUSSION THREAD

**Discussion by SDUTNB** [x-dr/chatgptProxyAPI#38] [Q&A]
pot翻译，自建和演示站都报错。余额查询Demo不翻墙也无法使用。是网络问题吗？

Http请求错误
Http Status: 403

**Comment by x-dr** [comment]
如果是第三个方法搭建的话 api 是 `https://xxxxx.pages.dev/api  `  看看是否填对；余额查询Demo不翻墙也可以使用的，pages.dev没被墙，如果不行大概是你网络问题
![image](https://github.com/x-dr/chatgptProxyAPI/assets/79109673/79ee84d4-12af-4eff-8639-3a48a67b2454)
